keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/diff/DualNumber.java

```diff
@@ -431,4 +431,13 @@ private DoubleTensor takeFromPartial(DoubleTensor from, int... indices) {
             .reshape(takeShape);
     }
 
+    @Override
+    public DualNumber doubleMinusThis(double that) {
+        throw new UnsupportedOperationException();
```

Comment:
As dual numbers are being thrown out in another PR, I didn't implement this here